src/Npgsql/NpgsqlCommand.cs

```diff
@@ -629,17 +629,19 @@ async Task PrepareLong()
                             default:
                                 throw connector.UnexpectedMessageReceived(msg.Code);
                             }
+
                             pStatement.CompletePrepare();
                             isFirst = false;
                         }
+                    }
```

Comment:
Braces here?

Reply:
Yes. Note that indentation was wrong previously, suggesting that the following code (Expect<ReadyForQueryMessage>) was in the loop when it wasn't. For big multiline cases, it seems better to have an explicit block even if not required.